src/material-experimental/mdc-checkbox/_checkbox-private.scss

```diff
@@ -34,13 +41,23 @@ $private-checkbox-theme-config: map.merge(mdc-checkbox-theme.$light-theme, (
     selected-hover-icon-color: $color,
     selected-icon-color: $color,
     selected-pressed-icon-color: $color,
-    unselected-focus-icon-color: $color,
-    unselected-hover-icon-color: $color,
+    unselected-focus-icon-color: $active-border-color,
+    unselected-hover-icon-color: $active-border-color,
 
     disabled-selected-icon-color: $disabled-color,
     disabled-unselected-icon-color: $disabled-color,
 
     unselected-icon-color: $border-color,
     unselected-pressed-icon-color: $border-color,
   ));
+
+  // We don't inherit the border focus style from MDC since we don't use their ripple.
+  // Instead we need to replicate it here.
+  .mdc-checkbox__native-control {
+    &:focus:enabled:not(:checked):not(:indeterminate):not([data-indeterminate=true]) {
```

Comment:
Isn't this covered by the `unselected-focus-icon-color` token?

Reply:
yes, but we don't seem to get those styles in our CSS, I think because they're tried up with the MDC ripple

Reply:
We have them already and they aren't bringing in any ripple styles.

Reply:
Here's where it should theoretically be getting emitted into our CSS: https://github.com/material-components/material-components-web/blob/master/packages/mdc-checkbox/_checkbox-theme.scss#L159, but something must be off between the DOM they expect us to have and what we actually have, because this style doesn't actually show up

Reply:
Yeah just traced that a little further and its because they're using some crazy ripple focus selector: https://github.com/material-components/material-components-web/blob/master/packages/mdc-ripple/_ripple-theme.scss#L486 I don't think we implement all of their ripple logic, so that doesn't actually show up for us

Reply:
Our selector is overly specific as well which will overrides difficult. Can it be simplified?

Reply:
No, I originally made it simpler, but it wasn't specific enough to override MDC's styles. At least the current level of specificity is required

Reply:
oof, I see now that MDC has the exact same selector. Should we at least move this into the structural styles and have it be based on a CSS variable? I've been introducing variables in a few places where the MDC component is switched over the variables as well (see https://github.com/angular/components/blob/main/src/material-experimental/mdc-radio/radio.scss#L41).

Reply:
ah good idea, then at least its easier to override. done!